vcs_server/Visualizer.py

```diff
@@ -27,8 +27,31 @@ def plot(self, prevWindowId, variable, template, method, width, height, opts={})
             plot.setGraphicsMethod(method)
             plot.setTemplate(template)
             all_vars = []
-            for obj in variable:
-                all_vars.append(cdms2.open(obj['uri'])(obj['variable']))
+            for varSpec in varSpecs:
+                f = cdms2.open(varSpec['uri'])
+                var = f(varSpec['variable'])
```

Comment:
do we want to read the whole variable in at this point? Shouldn't it be `var = f[varSpec['variable']]` i.e square brackets rather than regular brackets

Reply:
@doutriaux1 No we don't. So () reads the variable, [] does not? Are threre functions equivalents to this? I would rather use those if their effect is clearer. Thanks!